site/styles/_can-be-watched-button.scss

```diff
@@ -40,6 +40,7 @@
   }
 }
 
+/* stylelint-disable selector-max-compound-selectors */
```

Comment:
I'm surprised we haven't needed this in more places TBH... I wonder if it would be better to bump the limit up to accomodate instead, I mean we know we've got a problem.

Reply:
I'd rather keep the limit where it is to force any new css to conform.